js/nodes.js

```diff
@@ -37,9 +37,9 @@
     constructor(x, y) {
 
         if(x != undefined)
-            this.x = x
+            this.x = 1210
         if(y != undefined)
-            this.y = y
+            this.y = 25
```

Comment:
I would prefer this be based on the actual position of the create button (the HTML element's offsetLeft) rather than fixed coordinates. Try shrinking the browser window to the minimum height to see what I mean. The nodes will be created off-screen.

Reply:
gotcha